### DISCUSSION TITLE
Ohlc

### DISCUSSION THREAD

**Discussion by Inferno171** [alandefreitas/matplotplusplus#120] [Q&A]
Does it have OHLC plotting tool??

**Comment by alandefreitas** [comment]
Not out of the box. You need to combine other plot types to create an OHLC chart (lines and rectangles, or maybe there's an easier way with boxplots or bar plots). 
This is one of the first plot types I want to create after the OpenGL backend.
If you end up creating your own OHLC plot, please consider contributing. 🙂
